source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt

```diff
@@ -0,0 +1,60 @@
+.. _pipeline-builder-settings: 
+.. _change-pipeline-settings:
+.. _use-sample-mode-or-auto-preview:
+
+=====================================
+Aggregation Pipeline Builder Settings
+=====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+   
+You can adjust your |compass| Aggregation Pipeline Builder settings to customize 
+your aggregation experience and improve pipeline performance. 
+
+Settings
+--------
+
+To view and change your aggregation pipeline settings, click the gear icon at 
+the upper right of the pipeline builder to open the :guilabel:`Settings` panel. 
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 70 15 
+
+   * - Option 
+     - Description
+     - Default Value
+
+   * - Comment Mode 
+     - When enabled, adds helper comments to each stage. 
+     - Enabled 
+
+   * - Number of Preview Documents 
+     - Sets number of documents to show in the preview. 
```

Comment:
(nit)
Optional suggestion.
I know reference pages are often really sparse and that's by deign. In this case, I might add an article, plus an additional minor tweak.

```suggestion
     - Sets the number of documents to preview. 
```